src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/PopupRenderer.uwp.cs

```diff
@@ -0,0 +1,275 @@
+﻿using System;
+using System.ComponentModel;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.Platform.UWP;
+using UWPThickness = Windows.UI.Xaml.Thickness;
+using XamlStyle = Windows.UI.Xaml.Style;
+
+[assembly: ExportRenderer(typeof(BasePopup), typeof(PopupRenderer))]
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public class PopupRenderer : Flyout, IVisualElementRenderer
+	{
+		const double defaultBorderThickness = 2;
+		const double defaultSize = 600;
+		bool isDisposed = false;
+		XamlStyle flyoutStyle;
+		XamlStyle panelStyle;
+
+		public BasePopup Element { get; private set; }
+
+		internal ViewToRendererConverter.WrapperControl Control { get; private set; }
+
+		FrameworkElement IVisualElementRenderer.ContainerElement { get => null; }
+
+		VisualElement IVisualElementRenderer.Element { get => Element; }
+
+		public event EventHandler<VisualElementChangedEventArgs> ElementChanged;
+
+		public event EventHandler<PropertyChangedEventArgs> ElementPropertyChanged;
+
+		public PopupRenderer()
+		{
+		}
+
+		void IVisualElementRenderer.SetElement(VisualElement element)
+		{
+			if (element == null)
+				throw new ArgumentNullException(nameof(element));
+
+			if (!(element is BasePopup popup))
+				throw new ArgumentNullException("Element is not of type " + typeof(BasePopup), nameof(element));
+
+			var oldElement = Element;
+			Element = popup;
+			CreateControl();
+
+			Performance.Start(out var reference);
```

Comment:
You can remove this one as mentioned on iOS implementation.

Reply:
See https://github.com/xamarin/XamarinCommunityToolkit/pull/653#discussion_r538729419

Reply:
This will be removed in the next code push